docs/parameter_tables/HomSap/Africa_1B08.csv

```diff
@@ -1,6 +1,4 @@
 Population size,"7,778",Ancestral pop. size
-Population size,"25,636",Current pop. size
-Growth rate (per gen.),0.000175165,AFR pop. growth rate expansion
-Epoch Time (gen.),"6,809",Expansion time of ancestral pop.
-Generation time (yrs.),1,Generation time
+Population size,"25,636",AFR current pop. size
```

Comment:
I'd not say `AFR`, as that's the code from 1000 genomes; this is a different sample. Shouldn't this be "African" to match the populatiuon name?